lib/src/knobs/knobs.dart

```diff
@@ -42,33 +42,34 @@ abstract class KnobsBuilder {
   });
 
   /// Creates select field with [label], [initial] value and list of [options].
-  T options<T>(String label, {T initial, List<Option<T>> options = const []});
+  T? options<T>(String label, {T? initial, List<Option<T>> options = const []});
```

Comment:
Let's rather make it as `T` and use it without `?` and `!` everywhere.

First, using `!` for me is actually a code smell. Second, I'd say it's for the user to decide, whether they want nullable or non-nullable options (i.e. in your example with `MouseCursor`, the user should be free to choose `options<MouseCursor>` or `options<MouseCursor?>`).

Reply:
I updated the PR, have a look.

That makes the `initial` a required value, but I think that's fine. If you really want to use `null` then you have to pass it as an `initial` value. See the example for that. 

Now I wonder, if `options()` has required named parameters, should we change the API to only use named parameters? Like:

```dart
  T? options<T>({required String label, required T initial, List<Option<T>> options = const []});
```

That should be done for `boolean` and `text` too. What do you think? But that could also be part of a separate PR.

Reply:
Yeah, it makes sense to make it named as well. I'll merge this PR and will make it separately.